classes/submission/Repository.php

```diff
@@ -141,22 +141,22 @@ public function createDois(Submission $submission): array
 
     protected function mapDashboardViews(Collection $types, Context $context, User $user, bool $canAccessUnassignedSubmission, array $selectedRoleIds = []): Collection
     {
-        $views = parent::mapDashboardViews($types, $context, $user, $canAccessUnassignedSubmission, $selectedRoleIds);
+        $views = parent::mapDashboardViews($types->except([DashboardView::TYPE_REVIEW_EXTERNAL, DashboardView::TYPE_REVIEW_INTERNAL]), $context, $user, $canAccessUnassignedSubmission, $selectedRoleIds);
```

Comment:
I see that we are now showing submissions in both review stages at once for OMP. This means that `DashboardView::TYPE_REVIEW_INTERNAL` can be removed completely.